### PR TITLE
microsoftgraph/msgraph-sdk-java#211 fix ChunkedUpload limit

### LINKED ISSUE #211: ChunkedUploadProvider constructor size parameter int type limits upload to 2GB
The size parameter of constructor for ChunkedUploadProvider is int type which means that the maximum upload that can be done through the ChunkedUploadProvider is 2GB. But the maximum file size limit for OneDrive and OneDrive for Business is higher at 15GB:

https://support.office.com/en-us/article/invalid-file-names-and-file-types-in-onedrive-onedrive-for-business-and-sharepoint-64883a5d-228e-48f5-b3d2-eb39e07630fa#individualfilesize

Can this size parameter be changed to long so that uploads larger than 2GB is possible?

### Expected behavior
Be able to pass size parameter larger than max int in constructor of ChunkedUploadProvider.

### Actual behavior
Cannot pass size bigger than max int since the type of the parameter is int.

### Steps to reproduce the behavior
no steps required, the type of the parameter is currently int, it needs to be changed to long to make uploads larger than 2GB possible

### PR DESCRIPTION
**Pull request by SailReal**
Fixes #211

### Changes proposed in this pull request
- Fixes ChunkedUpload limit due to the type int, see #211

### Other links
- Please merge also https://github.com/microsoftgraph/msgraph-sdk-android/pull/107

### CODE REVIEW

File: src/main/java/com/microsoft/graph/concurrency/ChunkedUploadProvider.java [hunk 1]
```diff
@@ -82,7 +82,7 @@
     /**
      * The stream size
      */
-    private final int streamSize;
+    private final Long streamSize;
```
Comment (NakulSabharwal) [inline src/main/java/com/microsoft/graph/concurrency/ChunkedUploadProvider.java]: Please use long primitive type.

File: src/main/java/com/microsoft/graph/concurrency/ChunkedUploadProvider.java [hunk 2]
```diff
@@ -106,7 +106,7 @@
     public ChunkedUploadProvider(final UploadSession uploadSession,
                                  final IGraphServiceClient client,
                                  final InputStream inputStream,
-                                 final int streamSize,
+                                 final Long streamSize,
```
Comment (NakulSabharwal) [inline src/main/java/com/microsoft/graph/concurrency/ChunkedUploadProvider.java]: Please use long primitive type.

**Comment by hieudion** [comment]
Should we update the `beginIndex` as well to `long`? I tried with this in local and it is quite ok.
https://github.com/microsoftgraph/msgraph-sdk-java/pull/256/files#diff-584fe4861dea78fcff7ac4fc1acca325R74

**Comment by SailReal** [comment]
Thanks for your feedback. Should be fixed now!

**Pull request merged**